nrf_security/Kconfig

```diff
@@ -1476,6 +1476,16 @@ config MBEDTLS_SSL_CIPHERSUITES
 	  Warning: This field has offers no validation checks.
 	  MBEDTLS_SSL_CIPHERSUITES setting in mbed TLS config file.
 
+config NRF_SECURITY_DEFAULT_ENABLE_NONE
+	bool
```

Comment:
I have given this some more thoughts.
This deviates to much from how defaults are used elsewhere.
For NCS users, this becomes hard to understand when a default will be `y` and when it will be `n`.

If a user examines this file and reads:
```
default y if !NRF_SECURITY_DEFAULT_ENABLE_NONE
```
he has no way of predicting whether this code will default to `y` or `n`.

Compared to:
https://github.com/nrfconnect/sdk-zephyr/blob/dc1d8b4da3d58bbe5f4b8c290d2e1da3ce921cbd/subsys/bluetooth/Kconfig#L100
```
	default y if !BT_OBSERVER
```
where user can easily spot that this will be enabled when not making a Bluetooth observer.
or
https://github.com/nrfconnect/sdk-zephyr/blob/dc1d8b4da3d58bbe5f4b8c290d2e1da3ce921cbd/subsys/bluetooth/controller/Kconfig.ll_sw_split#L330
```
	default y if BT_HCI_RAW
```


It is not clear who might set `NRF_SECURITY_DEFAULT_ENABLE_NONE=y` and it is not a setting user can select for himself in a conf file.

menuconfig is not of much help:
![image](https://user-images.githubusercontent.com/9802583/95720256-4b5a9f00-0c71-11eb-9a0c-8b281af01b09.png)

If something should be done, then we should follow the principle of upstream.
Default everything to `n` and allow users a single `Enable all` for each group.
https://github.com/nrfconnect/sdk-zephyr/blob/master/modules/Kconfig.tls-generic#L46
```
config MBEDTLS_KEY_EXCHANGE_ALL_ENABLED
```
https://github.com/nrfconnect/sdk-zephyr/blob/master/modules/Kconfig.tls-generic#L106
```
config MBEDTLS_ECP_ALL_ENABLED
```